pkgs/lints/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
   - added [unnecessary_underscores] (https://github.com/dart-lang/core/issues/856)
```

Comment:
Uh...shouldn't this link be dropped then?

Reply:
The first link (bracketed) goes to the dart.dev docs for the lint; the 2nd (note the space) is the issue w/ the discussion of whether it should be added to the lint set.

Reply:
riiight